system-tests/src/test/java/org/eclipse/edc/samples/util/HttpRequestLoggerUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Initial implementation
+ *
+ */
+
+package org.eclipse.edc.samples.util;
+
+import org.testcontainers.containers.output.ToStringConsumer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+import java.util.List;
+
+import static org.eclipse.edc.samples.common.FileTransferCommon.getFileFromRelativePath;
+
+public class HttpRequestLoggerUtil {
```

Comment:
this util class is not useful, please give to `HttpRequestLoggerContainer` two constructors: the default one and one that will accept the log consumer, so in the tests it will be just a matter of instantiating it.